content/docs/command-reference/remove.md

```diff
@@ -47,61 +50,127 @@ how it helps replace data that is tracked by DVC.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example: remove a .dvc file
+## Example target: stage name
 
-Let's imagine we have `foo.csv` and `bar.csv` files, that are already
-[tracked](/doc/command-reference/add) by DVC:
+Let's imagine we have a `train` stage in `dvc.yaml`, and corresponding files in
+the <abbr>workspace</abbr>:
+
+```yaml
+train:
+  cmd: python train.py data.csv
+  deps:
+    - data.csv
+    - train.py
+  outs:
+    - logs
+    - model.h5
+```
 
 ```dvc
 $ ls
-bar.csv  bar.csv.dvc  foo.csv  foo.csv.dvc
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  model.h5  logs  train.py
+
 $ cat .gitignore
-/foo.csv
-/bar.csv
+/data.csv
+/model.h5
+/logs
 ```
 
-This removes `foo.csv.dvc` and double checks that its entry is gone from
-`.gitignore`:
+Using `dvc remove` on the stage name will remove that entry from `dvc.yaml`, and
+its outputs from `.gitignore`. With the `--outs` option, its outputs are also
+deleted (`logs` and `model.h5` in this example):
 
 ```dvc
-$ dvc remove foo.csv.dvc
+$ dvc remove train --outs
 
 $ ls
-bar.csv  bar.csv.dvc  foo.csv
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  train.py
+
 $ cat .gitignore
-/bar.csv
+/data.csv
 ```
 
-> The same procedure applies to tracked directories.
+> Notice that the dependencies (`data.csv` and `train.py`) are not deleted.
 
-## Example: remove a stage and its output
+## Example target: stage output files/directories
 
-Let's imagine we have a `train` stage in `dvc.yaml`, and corresponding files in
-the <abbr>workspace</abbr>:
+Assuming we have the same initial <abbr>workspace</abbr> as before:
 
 ```yaml
 train:
-  cmd: python train.py data.py
+  cmd: python train.py data.csv
   deps:
     - data.csv
     - train.py
   outs:
-    - model
+    - logs
+    - model.h5
 ```
 
 ```dvc
 $ ls
-dvc.lock  dvc.yaml  foo.csv  foo.csv.dvc  model  train.py
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  model.h5  logs  train.py
+
+$ cat .gitignore
+/data.csv
+/model.h5
+/logs
 ```
 
-Using `dvc remove` on the stage name will remove that entry from `dvc.yaml`, and
-its outputs from `.gitignore`. With the `--outs` option, its outputs are also
-deleted (just the `model` file in this example):
+`dvc remove` can also be used on **individual** <abbr>output</abbr> files or
+directories of a stage:
+
+```dvc
+$ dvc remove model.h5
+
+$ ls
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  logs  train.py
+
+$ cat .gitignore
+/data.csv
+/logs
+```
+
+The <abbr>output</abbr> file is actually being removed from the
```

Comment:
@daavoo besides removing it from the `.gitignore`, does it mean that we have pretty much `rm -f`?

Reply:
Apparently. Is that a bug @shcheklein ? It feels like it should be smart enough to edit dvc.yaml in this case.